clients/stellar-relay/src/connection/services.rs

```diff
@@ -277,6 +279,12 @@ pub(crate) async fn connection_handler(
 	loop {
 		match timeout(Duration::from_secs(connector.timeout_in_secs), actions_receiver.recv()).await
 		{
+			Ok(Some(ConnectorActions::Disconnect)) => {
+				log::error!("dropped");
```

Comment:
I would change this message to "Disconnected by dropping the TCP stream". And also, this does not have to be logged with 'error' level since it is intended.